site/content/guides/monitoring-as-code-cli.md

```diff
@@ -0,0 +1,347 @@
+
+  // click on the login button and go through the login procedure
+  await page.getByRole('button', { name: 'Log in' }).click();
+  await page.getByPlaceholder('Email').fill('user@email.com');
+  await page.getByPlaceholder('Password').fill('supersecure1');
+  await page.getByRole('button', { name: 'Sign In' }).click();
+
+  // wait until the login Welcome back message is shown
+  await page.getByText('Welcome back, user@email.com').isVisible();
+});
+```
+{{< /tab >}}
+{{< tab "Search" >}}
+ ```js
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
+
+  const bookList = [
+    "The Foreigner",
+    "The Transformation",
+    "For Whom the Ball Tells",
+    "Baiting for Robot",
+  ];
+
+  // navigate to our target web page
+  await page.goto("https://danube-web.shop/");
+
+  // search for keyword
+  await page.click(".topbar > input");
+  await page.type(".topbar > input", "for");
+  await page.click("#button-search");
+  await page.waitForSelector(
+    ".shop-content > ul > .preview:nth-child(1) > .preview-title"
+  );
+
+  // halt immediately if results do not equal expected number
+  let resultsNumber = (await page.$$(".preview-title")).length;
+
+  await expect(resultsNumber).toEqual(bookList.length);
+
+  // remove every element found from the original array...
+  for (var i = 0; i < resultsNumber; i++) {
+    const resultTitle = await page.$eval(
+      `.preview:nth-child(${i + 1}) > .preview-title`,
+      (e) => e.innerText
+    );
+
+    const index = bookList.indexOf(resultTitle);
+    bookList.splice(index, 1);
+  }
+
+  // ...then assert that the original array is now empty
+  await expect(bookList.length).toEqual(0);
+
+});
+```
+{{< /tab >}}
+{{< tab "Checkout" >}}
+ ```js
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
+  // navigate to our target web page
+  await page.goto('https://danube-web.shop/');
+  
+  // add the first item to the cart
+  await page.getByText('Haben oder haben').click();
+  await page.getByRole('button', { name: 'Add to cart' }).click();
+
+  // navigate to cart and proceed
+  await page.getByRole('button', { name: 'Checkout' }).click();
+
+  // fill out checkout info
+  await page.locator('#s-name').fill('Max');
+  await page.getByPlaceholder('Surname').fill('Mustermann');
+  await page.getByPlaceholder('Address').fill('Charlottenstr. 57');
+  await page.getByPlaceholder('Zipcode').fill('10117');
+  await page.getByPlaceholder('City').fill('Berlin');
+  await page.getByPlaceholder('Company (optional)').fill('Firma GmbH');
+  await page.getByLabel('as soon as possible').check();
+
+  // confirm checkout
+  await page.getByRole('button', { name: 'Buy' }).click();
+
+  // wait until the order confirmation message is shown
+  await page.getByText('All good, order is on the way. Thank you!! Shop more').isVisible();
+});
+ ```
+{{< /tab >}}
+{{< /tabs >}}
+
+Let's start off by creating a brand new folder:
```

Comment:
This section is hard to copy & paste.

Reply:
I assume you mean these commands: 'mkdir checkly-mac-cli-example && cd $_' ?

Reply:
Yes, it's easier to just have them as separate lines, instead of inline code items

Reply:
Yeah, I would love to have proper copy&paste buttons. But I'm ignoring that desire, knowing we are already working on a new website. I'm adding some new lines for now.